examples/Db2/README.md

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
Looks like this file was copied from the module/db2/README.md file.  Need to pattern it after https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/blob/main/examples/cp4i/README.md.